tests/routes_tests/test_report.py

```diff
@@ -286,16 +286,16 @@ def test_two_round_audit_report(client, election_id):
 
     # We'll just test a sampling of lines that should include a good variety of cases
     assert ballot_lines[:10] == [
-        f'"Batch 1, #111",Round 2: 0.034167626,Audited,{candidate_id_1},Comment for ballot 0',
+        f'"Batch 1, #111",Round 2: 0.034167626,Audited,{candidate_id_2},',
         f'"Batch 1, #122",Round 1: 0.012066605,Audited,{candidate_id_1},',
         f'"Batch 10, #10",Round 1: 0.010939432,Audited,{candidate_id_1},Comment for ballot 0',
         f'"Batch 10, #103",Round 2: 0.031357473,Audited,{candidate_id_1},',
-        '"Batch 10, #151",Round 1: 0.012381762,Audited,Blank vote/no mark,Comment for ballot 3',
-        '"Batch 10, #175",Round 2: 0.021956866,Audited,Audit board can\'t agree,',
+        f'"Batch 10, #151",Round 1: 0.012381762,Audited,Blank vote/no mark,Comment for ballot 3',
+        f'"Batch 10, #175",Round 2: 0.021956866,Audited,{candidate_id_2},Comment for ballot 1017',
         f'"Batch 10, #200",Round 1: 0.000030407,Audited,{candidate_id_2},',
         f'"Batch 10, #59",Round 1: 0.002728647,Audited,{candidate_id_1},',
-        '"Batch 10, #72",Round 1: 0.009650515,Audited,Audit board can\'t agree,',
-        '"Batch 100, #106",Round 2: 0.015314474,Audited,Blank vote/no mark,Comment for ballot 3',
+        f'"Batch 10, #72",Round 1: 0.009650515,Audited,Audit board can\'t agree,',
+        f'"Batch 100, #106",Round 2: 0.015314474,Audited,{candidate_id_2},',
```

Comment:
Same here, changed by adding the db unique constraint. I confirmed this by adding the db constraint on master and running the tests.

Reply:
same comment as above: I suggest having an implementation-independent ordering that is explicitly requested by the SQL query, so changes in indexing and other internal optimizations don't break these tests.

Reply:
Added here: 540f5f2dcb003ed68a068173fd16325f483a31ed

Reply:
Turns out this same call to `Jurisdiction.audit_boards` was at the root of both of these cases